### DISCUSSION TITLE
posts/giscus-%EC%84%A4%EC%A0%95/

### DISCUSSION THREAD

**Discussion by giscus** [Jihwan98/Jihwan98.github.io#3] [General]
# posts/giscus-%EC%84%A4%EC%A0%95/

✅ 댓글 utterances $\rightarrow$ giscus로 변경

https://jihwan98.github.io/posts/giscus-%EC%84%A4%EC%A0%95/

**Comment by Jihwan98** [comment]
댓글 테스트

**Comment by wlsgjd** [comment]
덕분에 쉽게 세팅했습니다. 감사합니다.

**Reply by Jihwan98** [reply]
도움이 되어서 기쁘네요 !!
